cookbook/contract-build-meta.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Add meta data to contract WASM on build
+hide_table_of_contents: true
+description: Include meta data in the contract WASM byte code on build
+custom_edit_url: https://github.com/stellar/stellar-cli/edit/main/cookbook/contract-build-meta.mdx
+---
+
+To build a smart contract, use the `stellar contract build` command. The `--meta` option will allow you to add additional meta data entries to the `contractmetav0` custom section of the resulting WASM file at build time.
+
+```bash cookbooktest.ignore
+export SHA=$(git rev-parse HEAD)
```

Comment:
i put this in a separate bash block, so that we could ignore it from the cookbook test.  

otherwise, the way the cookbook test is written, i think that it would interpret this as an arg to the stellar command, and fail.